github-build.mjs

```diff
@@ -0,0 +1,58 @@
+import pkg from './package.json'
+import Builder from 'alpheios-node-build'
+import generateBuildInfo from './node_modules/alpheios-node-build/dist/support/build-info.mjs'
+import { execFileSync, execSync } from 'child_process'
+import * as core from '@actions/core'
+
+(async function() {
+  const buildDT = Date.now()
+  const buildInfo = generateBuildInfo(buildDT)
+  console.log(`Starting build ${buildInfo.name}`)
+
+  const baseVersion = pkg.version.split('-')[0]
+  const version = `${baseVersion}-${buildInfo.name}`
+  console.log(`Setting a package version to ${version}`)
+  let output
+  try {
+    if (buildInfo.branch === 'qa' || buildInfo.branch === 'production') {
+      console.info(`Installing alpheios-core`)
+      output = execSync(`npm install https://github.com/alpheios-project/alpheios-core#${buildInfo.branch}`)
+    }
+  } catch (error) {
+    console.error('Components install failed:', error)
+    process.exit(1)
+  }
+  try {
+    const output = execSync(`npm version ${version} --no-git-tag-version --force`, { encoding: 'utf8' })
+  } catch (e) {
+    console.error('Cannot execute npm version:', e)
+    process.exit(1)
```

Comment:
Here is the same return code as before. Should we use a unique one?

Reply:
Yes, this is the way it was in tagged-commit but I'll go ahead and fix it.